### DISCUSSION TITLE
Change commit message made by TiddlyWiki using GitHub Saver?

### DISCUSSION THREAD

**Discussion by JaegerStein** [TiddlyWiki/TiddlyWiki5#5886] [Q&A]
TiddlyWiki saves all commits with the message "Saved by TiddlyWiki", which is quite unhelpful when trying to find an older commit in case I want to look up something that has since been deleted.

I would prefer it to look something like "Created X, Edited Y, Deleted Z", with the specific tiddlers mentioned in the description. Is this possible?

**Comment by pmario** [comment]
You can change this tiddler: https://tiddlywiki.com/#%24%3A%2Flanguage%2FControlPanel%2FSaving%2FGitService%2FCommitMessage and then save.

**Comment by Jermolene** [comment]
Hi @JaegerStein as @pmario notes the tiddler `$:/language/ControlPanel/Saving/GitService/CommitMessage` contains the string used for the commit message. However, `github.js` (and `gitea.js` and `gitlab.js`) uses `$tw.langugage.getRawString()` to retrieve the string, which means that it does not get wikified. That makes it impossible to use wikitext or macros within the commit message.

I think a worthwhile improvement would be for the core to wikify those commit messages, so that it would be possible to set the message to things like `Committed by {{$:/status/UserName}} at <<now>>`.

https://github.com/Jermolene/TiddlyWiki5/blob/cb726f40fa83c6c6676ec43188eeb2ddc55b4d29/core/modules/savers/github.js#L72
